specification/redis/Cache.Management/RedisResource.tsp

```diff
@@ -0,0 +1,228 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+
+namespace Microsoft.Cache;
+/**
+ * A single Redis item in List or Get Operation.
+ */
+model RedisResource
+  is Azure.ResourceManager.TrackedResource<RedisProperties, false> {
+  ...ResourceNameParameter<
+    Resource = RedisResource,
+    KeyName = "cacheName",
+    SegmentName = "redis",
+    NamePattern = "^([a-zA-Z0-9][a-zA-Z0-9-]*[a-zA-Z0-9]|[a-zA-Z0-9])$"
+  >;
+  ...Azure.ResourceManager.AvailabilityZonesProperty;
+
+  /**
+   * The identity of the resource.
+   */
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-invalid-envelope-property" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  identity?: Azure.ResourceManager.CommonTypes.ManagedServiceIdentity;
```

Comment:
What's wrong with this?
identity is normally a top-level property right?
Or should we be inheriting it from TrackedResource??

Reply:
Normally, this would just be a mix-in of ManagedServiceIdentityProperty, why not in this case?